test/global-rate-limit.test.js

```diff
@@ -358,13 +357,16 @@ test('With redis store', async t => {
   await sleep(1100)
 
   res = await fastify.inject('/')
-  redis.flushall(noop)
-  redis.quit(noop)
 
   t.equal(res.statusCode, 200)
   t.equal(res.headers['x-ratelimit-limit'], 2)
   t.equal(res.headers['x-ratelimit-remaining'], 1)
   t.equal(res.headers['x-ratelimit-reset'], 1)
+
+  t.teardown(async () => {
+    await redis.flushall()
+    await redis.quit()
+  })
 })
 
 test('Skip on redis error', async t => {
```

Comment:
This test is doing what it suppose to do. We need another way to simulate Redis error.

Reply:
I patched it now according to @Eomm remarks. 

Is it not acceptable?

Reply:
I believe we are just commenting on the same thing.
The patch appear later than my comment.